RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -0,0 +1,63 @@
+# This is meant for items that have a date of 2023
+
+boolean auto_haveRockGarden()
+{
+	static item rockGarden = $item[Packet Of Rock Seeds];
+	return auto_is_valid(rockGarden) && auto_get_campground() contains rockGarden;
+}
+
+void rockGardenEnd() //broke these out so they aren't handled at the start of everyday but ASAP after numberology
+{
+	//while we will probably never get these automatically, should handle them anyway
+	if (item_amount($item[Molehill Mountain]) > 0 && !get_property("_molehillMountainUsed").to_boolean()){
+		use(1,$item[Molehill Mountain]);
+	}
+
+	if((item_amount($item[strange stalagmite]) > 0) && !get_property("_strangeStalagmiteUsed").to_boolean())
+	{
+		use(1,$item[strange stalagmite]);
+	}
+	return;
+}
+
+void pickRocks()
+{
+	//Pick rocks everyday
+	//If we manage to get a lodestone, will not use it, because it is a one-a-day and user may want to use it in specific places
+	//
+    if (!auto_haveRockGarden()) return;
+	visit_url("campground.php?action=rgarden1");
+	if(get_property("desertExploration").to_int() < 100)
+	{
+		visit_url("campground.php?action=rgarden2");
+	}
+	visit_url("campground.php?action=rgarden3");
+	return;
+}
+
+boolean auto_haveSITCourse()
+{
+	static item sitCourse = $item[S.I.T. Course Completion Certificate];
+	return auto_is_valid(sitCourse) && item_amount(sitCourse) > 0;
+}
+
+void auto_SITCourse()
+{
+	if (!auto_haveSITCourse()) return;
+	//Best choice seems to be insectologist
+	if (!have_skill($skill[insectologist])){
```

Comment:
Many of my comments are to use buffs from the other 2. Thinking is if we ever decide a different one is better, it will be easy to switch as they are already supported. It is extra work though so I'd still approve even if you don't support the other courses fully